shared/middleware/helpers/null_pub_sub_api.rb

```diff
@@ -0,0 +1,11 @@
+# Interface to fake interactions with a PubSub system, like Pusher
+# Should respond to same messages as PusherApi
+
+class NullPubSubApi
+  def self.configure_keys
```

Comment:
Removed the `configure_keys` method.  This is static initialization, and can be done outside of a method in the class definition, oddly enough. Ruby is weird.